Mathlib/Topology/Algebra/Module/LocallyConvex.lean

```diff
@@ -145,6 +145,13 @@ theorem Disjoint.exists_open_convexes [LocallyConvexSpace 𝕜 E] {s t : Set E}
   simp_rw [UniformSpace.ball, ← preimage_comp, sub_eq_neg_add] at hV
   exact hV
 
+/-- In a locally convex space, if `x ∉ s` and `s` is convex and closed, then we can find open
+disjoint convex sets containing them. -/
```

Comment:
"containing them" here is confusing, presumably you mean that one contains x and the other contains s?

Reply:
I was following the existing docstring, which I've now modified accordingly

Reply:
Right, and while the existing docstring made sense as written thanks to the symmetry, the new one didn't!